full-stack-tests/ui/tests/popout-widget.test.ts

```diff
@@ -220,7 +220,9 @@ test.describe("popout widget", () => {
       WidgetState.Floating
     );
     await expect.poll(async () => popoutPage.isClosed()).toBe(true);
-    expect(mountCount).toBe(2);
-    expect(unMountCount).toBe(1);
+    // Due to the change to Popouts where `WidgetContentContainer` is displayed instead of a React Node. The actual widget is no longer unmounted
+    // and is simply sent to a different `WidgetContentContainer`. That's why the unmount count was changed to 0 here. TODO: Delete this test?
```

Comment:
We can keep the test as this is now what will be expected from now on, however we should probably rename the test.